.gitlab/jobs/tioga.yml

```diff
@@ -0,0 +1,32 @@
+#########################################################################
+#
+# This file is part of the SAMRAI distribution.  For full copyright
+# information, see COPYRIGHT and LICENSE.
+#
+# Copyright:     (c) 1997-2023 Lawrence Livermore National Security, LLC
+#
+#########################################################################
+
+variables:
+  BUILD_OPTION: "--build-only"
```

Comment:
It looks like this --build-only flag doesn't end up in the command, so the CI attempted to run the tests for this configuration that is supposed to be only a build.